platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/gossip/chatter/ChatterGossip.java

```diff
@@ -103,36 +103,37 @@ public class ChatterGossip extends AbstractGossip {
     private final List<StoppableThread> chatterThreads = new LinkedList<>();
     private final ChatterEventMapper chatterEventMapper = new ChatterEventMapper();
     private final SequenceCycle<EventIntakeTask> intakeCycle;
-    private final Clearable clearAllPipelines;
+    private final Clearable clearAllInternalPipelines;
```

Comment:
Consider moving this member variable up to the abstract class. I took a look, and it seems to me like it would work. I might have missed something, though

Doing this would reduce # of member variables, and allow child classes to avoid implementing `clear()`

Reply:
I'd like to, but it's not so simple to do. Java doesn't let us construct things prior to calling super, and there are objects we need that are constructed later that need to go into the construction of `clearAllInternalPipelines`.